calculator_test.go

```diff
@@ -2,23 +2,558 @@ package calculator_test
 
 import (
 	"calculator"
+	"fmt"
+	"math"
+	"math/rand"
 	"testing"
 )
 
+func closeEnough(
+	roundResult, divisionResult, tolerance float64,
```

Comment:
These names are too long, as evidenced by the fact that you've had to wrap the function signature across multiple lines! This function is only concerned with comparing two numbers—it doesn't matter in the least what they represent, at least not here—so we might use `a` and `b`, for example.

Reply:
Fixed in f292791